starcheck/calc_ccd_temps.py

```diff
@@ -195,10 +201,22 @@ def get_interval_temps(intervals, times, ccd_temp):
         stop_idx = 1 + np.searchsorted(times, interval['tstop'])
         start_idx = -1 + np.searchsorted(times, interval['tstart'])
         ok_temps = ccd_temp[start_idx:stop_idx]
+        itimes = times[start_idx:stop_idx]
```

Comment:
Use consistent naming for like-defined objects, which in this case would be `ok_times`.  Or you could name both even more clearly with `interval_*`.  Usually `ok` is just reserved for a single one-off boolean mask.